### DISCUSSION TITLE
Is there a support for React Native and ReactJs ?

### DISCUSSION THREAD

**Discussion by IamDhana** [processone/ejabberd#3608] [Questions] [answered]
Hi All,

I'm planning to add a **chat** application in my project for both mobile and web application based on **React Native and React Js**.

Will ejabberd support these platforms?

If yes can you guys share documentation?

I have searched the internet I didn't get any useful link.

**Comment by badlop** [comment, marked as answer]
From what I read, React Native is a framework to develop mobile applications.

If you plan to develop in React Native a XMPP client that will connect to ejabberd (or to any other XMPP server), then your question should be: **Is there any library to develop a XMPP client in React Native?**

If you search "xmpp react native", you will find some libraries. If you use a good library, your client should work correctly with ejabberd and any other XMPP/Jabber server.
